### ISSUE TITLE
Detected by defender, but only after bind

### ISSUE THREAD

**Issue by Bartixxx32** [UnamSanctam/UnamBinder#25]
As in topic, miner util not binded its undetected by defender but, when binded. It is. Is anything i can do for it?

**Comment by UnamSanctam** [comment]
Yes of course, the binder generates its own .exe so that will be what gets scanned and not the miner, the current binder is detected (you can see when the last release was) unless you crypt it, but I'm also almost done with the new update which will reduce detections.

**Comment by Bartixxx32** [comment]
Great to hear! Thanks

**Comment by harry133712** [comment]
new binder also gets detected unfortunately

**Comment by UnamSanctam** [comment]
Yes you'll need to obfuscate or crypt it, it's been undetected during 4 days of (final) testing but obviously it gets detected eventually (it's been out for 4 hours so most likely there's been by far over 10 builds uploaded to VirusTotal and similar sites).

**Comment by harry133712** [comment]
i see will try some obfuscators to get it undetected thanks for ur response

**Comment by PandoricGalaxy** [comment]
Will .net obfuscators work to crypt/obfuscated the generated exe?

**Comment by UnamSanctam** [comment]
Yes if you choose to build it as "Managed".

**Comment by PandoricGalaxy** [comment]
Alr8 thx
